examples/mqtt_client/mqtt_client.c

```diff
@@ -0,0 +1,249 @@
+
+#include "espressif/esp_common.h"
+#include "esp/uart.h"
+
+#include <string.h>
+
+#include <FreeRTOS.h>
+#include <task.h>
+#include <ssid_config.h>
+
+#include <espressif/esp_sta.h>
+#include <espressif/esp_wifi.h>
+
+#include <paho/MQTTESP8266.h>
+#include <paho/MQTTClient.h>
+
+#include <semphr.h>
+
+#define MQTT_HOST ("192.168.1.4")
```

Comment:
You don't have to change this, but in you're interested the mosquitto project runs a publically available MQTT server at http://test.mosquitto.org/ . Would allow people to run the example without needing their own server?

(If you change it, please put a comment with a link to http://test.mosquitto.org/ so people can see what the server is for.)